com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree8Passes.hlsl

```diff
@@ -378,7 +378,10 @@ half4 SpeedTree8Frag(SpeedTreeFragmentInput input) : SV_Target
     half4 diffuse = SampleAlbedoAlpha(uv, TEXTURE2D_ARGS(_MainTex, sampler_MainTex)) * _Color;
 
     half alpha = diffuse.a * input.interpolated.color.a;
-    AlphaDiscard(alpha, 0.3333);
+
+#if defined(_ALPHATEST_ON)
```

Comment:
Previously the `#if defined(_ALPHATEST_ON)` was in the inside of the function to not require explicit `#ifdef` soup outside.
I personally like it explicit, so you instantly know if it's done or not, but it's inconsistent how many similar functions work in URP shaders.
Also potentially breaking user code (bu that's already broken by the API change.)